app/views/layouts/sql_probe/sql_probe.html.erb

```diff
@@ -1,23 +1 @@
-<!DOCTYPE html>
-<html>
-<head>
-  <meta charset='utf-8' />
-  <meta name="viewport" content="width=device-width, initial-scale=1" />
-  <meta name="apple-mobile-web-app-capable" content="yes" />
-  <title>SQL Probe</title>
-  <script>window.SqlProbe = <%= raw javascript_global_config.to_json %></script>
-  <%= stylesheet_link_tag    "sql_probe/application", media: "all" %>
-  <%= javascript_include_tag "sql_probe/application" %>
-  <%= csrf_meta_tags %>
-  <%= yield :head %>
-</head>
-<body>
-
-<%= render 'layouts/sql_probe/navbar' unless @disable_top_navbar %>
-
 <%= yield %>
-
-<%= yield :javascript %>
-<%= render 'layouts/sql_probe/ga' if Rails.env.production? %>
```

Comment:
I see what you did. And did things more Railsy. Commit: 0eccf09

Basically, I put back the layout page, and told the live controller to ignore it.